lms/data_tasks/report/create_from_scratch/02_entities/03_events/01_create_view.sql

```diff
@@ -23,7 +23,7 @@ CREATE MATERIALIZED VIEW report.events AS (
 
         translated_events AS (
             SELECT
-                DATE_TRUNC('week', timestamp)::date AS timestamp_week,
+                report.multi_truncate('week', timestamp) AS timestamp_week,
```

Comment:
Use the common function on these instead of doing the `-1 day::interval` all over the place.